### PR TITLE
set at least some EDITOR value...

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

File: scripts/mkcloud [hunk 1]
```diff
@@ -229,7 +229,7 @@ EOF
     env|grep -e ^debug_ -e ^pre_ -e ^want_ -e ^net_ -e ^nodenumber -e ^clusterconfig | sort >> $mkcconf
 
     $scp $qa_crowbarsetup $mkcconf root@$adminip:
-    ssh $sshopts root@$adminip "echo `hostname` > cloud ; . qa_crowbarsetup.sh ; $@"
```
Comment (bmwiedemann) [inline scripts/mkcloud]: IMHO, crowbar (and other tools using $EDITOR) should have a sane default such as EDITOR=edit

Comment (jsuchome) [inline scripts/mkcloud]: I understand there could be better solution, but this one would also help and is quick and easy

**Comment by rsalevsky** [comment]
+1

**Comment by mimi1vx** [comment]
i think in SLE is default value for EDITOR=vi

**Comment by mjura** [comment]
+1

**Comment by bmwiedemann** [comment]
This code needs to go into qa_crowbarsetup.sh as it needs to run on the admin node. Otherwise, it will be missing in the physical clouds. You could add it to the prepareinstcrowbar step

**Comment by bmwiedemann** [comment]
actually we already have
```
root@crowbar:~ # cat /etc/profile.d/editor.sh
EDITOR=vi
root@crowbar:~ # rpm -qf /etc/profile.d/editor.sh
file /etc/profile.d/editor.sh is not owned by any package
```
but it is not exported, so is not used by sub-processes.

The editor.sh comes from chef/cookbooks/provisioner/recipes/base.rb so it needs to be fixed in there.

**Comment by bmwiedemann** [comment]
moved to https://github.com/crowbar/barclamp-provisioner/pull/440

**Pull request closed without merging**